### ISSUE TITLE
A note whose length is specified in seconds cannot appear at the end of a sequence

### ISSUE THREAD

**Issue by UlyssesZh** [alda-lang/alda#381]
# 🐞 Bug report 🐞

## Description
A note whose length is specified in seconds cannot appear at the end of a sequence.
The bug does not appear in Alda 1 (the same reproducing steps do not reproduce the bug in Alda 1.5.0).
The bug cannot be reproduced if the length is specified in milliseconds.

## Steps to Reproduce

```
alda play -c "[c1s]"
```

## Expected Behavior

Play without errors.

## Actual Behavior

```
<no file>:1:5 Unexpected ']' in note/rest/name
```

## Environment

**Operating system and version:**
Windows 10

**Alda version:**

```
$ alda version
alda 2.0.1
```

```
$ alda-player info
alda-player 2.0.1
log path: C:\Users\ulysses\AppData\Local\alda\cache\logs
```

**Health check:**

```
$ alda doctor
OK  Parse source code
OK  Generate score model
OK  Find an open port
OK  Send and receive OSC messages
OK  Locate alda-player executable on PATH
OK  Check alda-player version
OK  Spawn a player process
OK  Ping player process
OK  Play score
OK  Export score as MIDI
OK  Locate player logs
OK  Player logs show the ping was received
OK  Shut down player process
OK  Spawn a player on an unknown port
OK  Discover the player
OK  Ping the player
OK  Shut the player down
OK  Start a REPL server
nREPL server started on port 63288 on host localhost - nrepl://localhost:63288
OK  Interact with the REPL server
```

**Logs:**

No useful logs about this.

**Comment by daveyarwood** [comment]
Nice catch! I'll work on fixing this soon.

**Comment by daveyarwood** [comment]
Fixed in d766a75! Will include this fix in the next release.

**Comment by daveyarwood** [comment]
I've just released Alda 2.0.3, which includes a fix for this issue.

Thanks for reporting this!
